Fix Failing Tests on Development

Seems we had some silent failures on `development`. There is no obvious soloution here. The function `BabeApi_configuration` exists in the blob (verified through wasm2wat). The other failure is related to #360 

Some of the failures:
```go
=== RUN   TestMakeNode
--- FAIL: TestMakeNode (0.49s)
=== RUN   TestMakeNode/node_from_config_(norpc)
t=2019-11-26T16:41:09-0500 lvl=info msg="🕸\t Initializing node" name=gossamer id=gossamer protocolID=gossamer bootnodes=[/ip4/104.211.54.233/tcp/30363/p2p/16Uiu2HAmFWPUx45xYYeCpAryQbvU3dY8PWGdMwS2tLm1dB1CsmCj]
t=2019-11-26T16:41:09-0500 lvl=info msg="Database closed"
t=2019-11-26T16:41:09-0500 lvl=info msg="Database closed"
t=2019-11-26T16:41:09-0500 lvl=info msg="🕸\t Configuring node..." datadir=/home/d/dev/gossamer/cmd/gossamer/test_data protocolID=gossamer bootnodes=[]
    --- FAIL: TestMakeNode/node_from_config_(norpc) (0.14s)
        configcmd_test.go:431: could not find exported function BabeApi_configuration
=== RUN   TestMakeNode/default_node_(norpc)
t=2019-11-26T16:41:09-0500 lvl=info msg="🕸\t Initializing node" name=gossamer id=gossamer protocolID=gossamer bootnodes=[/ip4/104.211.54.233/tcp/30363/p2p/16Uiu2HAmFWPUx45xYYeCpAryQbvU3dY8PWGdMwS2tLm1dB1CsmCj]
t=2019-11-26T16:41:10-0500 lvl=info msg="Database closed"
t=2019-11-26T16:41:10-0500 lvl=info msg="Database closed"
t=2019-11-26T16:41:10-0500 lvl=info msg="🕸\t Configuring node..." datadir=/home/d/.gossamer protocolID=gossamer bootnodes=[]
    --- FAIL: TestMakeNode/default_node_(norpc) (0.19s)
        configcmd_test.go:431: could not find exported function BabeApi_configuration
=== RUN   TestMakeNode/default_node_(rpc)
t=2019-11-26T16:41:10-0500 lvl=info msg="🕸\t Initializing node" name=gossamer id=gossamer protocolID=gossamer bootnodes=[/ip4/104.211.54.233/tcp/30363/p2p/16Uiu2HAmFWPUx45xYYeCpAryQbvU3dY8PWGdMwS2tLm1dB1CsmCj]
    --- FAIL: TestMakeNode/default_node_(rpc) (0.10s)
        configcmd_test.go:426: Cannot acquire directory lock on "/home/d/.gossamer/state".  Another process is using this Badger database.: resource temporarily unavailable

```

--- a/cmd/gossamer/configcmd_test.go
+++ b/cmd/gossamer/configcmd_test.go
@@ -84,8 +84,8 @@ func createCliContext(description string, flags []string, values []interface{}) 
 	return context, nil
 }
 
-const TESTS_FP string = "../../runtime/test_wasm.wasm"
-const TEST_WASM_URL string = "https://github.com/ChainSafe/gossamer-test-wasm/blob/c0ff6e519676affd727a45fe605bc7c84a0a536d/target/wasm32-unknown-unknown/release/test_wasm.wasm?raw=true"
+const TESTS_FP string = "../../substrate_test_runtime.compact.wasm"
+const TEST_WASM_URL string = "https://github.com/noot/substrate/blob/add-blob/core/test-runtime/wasm/wasm32-unknown-unknown/release/wbuild/substrate-test-runtime/substrate_test_runtime.compact.wasm?raw=true"
 
 // Exists reports whether the named file or directory exists.
 func Exists(name string) bool {

